pulser/register/special_layouts.py

```diff
@@ -166,3 +170,6 @@ def __str__(self) -> str:
             f"TriangularLatticeLayout({self.number_of_traps}, "
             f"{self._spacing}µm)"
         )
+
+    def _to_dict(self) -> dict[str, Any]:
+        return obj_to_dict(self, self.number_of_traps, self._spacing)
```

Comment:
Warning: it seems the only way the parameter `n_traps`'s type is "checked" is inside `triangular_hex` with `if n_points < 7`

Reply:
Do you think this is an issue? Do you want me to make sure that it's an `int` with `int(n_traps)`?

Reply:
I think it's an issue because `TriangularLatticeLayout` is included in `SUPPORTED_MODULES` and thus is exposed to the outside. We probably want to check that users won't inject something else than an `int`.

Reply:
Ok, but note that what they inject there still needs to be in the supported modules, so I don't think they can put anything harmful. 

Still, how do you propose we make this check?